src/integration-tests/integration-test.py

```diff
@@ -826,6 +833,9 @@ def check_logs_for_errors(self) -> None:
                 self.logger.error("error log: %s", message)
             seen_errors = True
 
+        if self.ignore_log_errors:
```

Comment:
i am not trying to customize the errors that we ignore. 
The idea is to prevent cases when an error when running a job prevents us from running other tests